src/frontend/.eslintrc.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
     "plugin:react/recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:sonarjs/recommended",
-    "plugin:jest-playwright/recommended",
```

Comment:
why did you remove this linter plugin?

Reply:
It has a dependency on jest-playwright runner which we are trying to switch-off